src/shop-cart-item.html

```diff
@@ -14,6 +14,7 @@
 <link rel="import" href="shop-icons.html">
 <link rel="import" href="shop-image.html">
 <link rel="import" href="shop-select.html">
+<link rel="import" href="shop-md-decorator.html">
```

Comment:
Couldn't this be added as a lazy-import which won't actually require the fake elements to exist.